src/models/network/NetworkAppController.js

```diff
@@ -52,23 +93,26 @@ export default class NetworkAppController extends NetworkBaseController {
     return this.app.unsetImplementation(contractAlias);
   }
 
-  async createProxy(contractAlias, initMethod, initArgs) {
+  async createInstance(contractAlias, upgradeable, initMethod, initArgs) {
     await this.fetch();
     const contractClass = this.localController.getContractClass(contractAlias);
-    this.checkInitialization(contractClass, initMethod, initArgs);
-    const proxyInstance = await this.app.createProxy(contractClass, contractAlias, initMethod, initArgs);
-    const implementationAddress = await this.app.getImplementation(contractAlias);
-    this._updateTruffleDeployedInformation(contractAlias, proxyInstance)
+    this.checkInitialization(contractClass, initMethod);
+    const instance = upgradeable
+      ? await this.app.createProxy(contractClass, contractAlias, initMethod, initArgs)
+      : await this.app.createNonUpgradeableInstance(contractClass, contractAlias, initMethod, initArgs)
```

Comment:
I'd consider pushing this ternary operation down to zos-lib `App`, so this becomes:
```js
const proxyInstance = await this.app.createInstance(upgradeable, contractClass, contractAlias, initMethod, initArgs);
```

Reply:
I like the idea, in the meantime, I pushed this PR in order to have this logic in `zos-lib`:  https://github.com/zeppelinos/zos-lib/pull/209